### ISSUE TITLE
Ainda funciona no macOS Catalina, em 2019...

### ISSUE THREAD

**Issue by imerso** [lailsonbm/ABNT2-Layout#3]
Só queria dizer, para quem estiver procurando, que funciona no Catalina em 2019... boa sorte.

**Comment by otonii** [comment]
Pra mim a botão da contra barra e aspas estão invertidos. Mas o resto está funcionando normalmente.

**Comment by gabfr** [comment]
Sim, obotão da contra barra e aspas estão invertidos. Como você fez pra consertar isso @otonii?

**Comment by gabfr** [comment]
Descobri uma solução, segue: https://www.insanelymac.com/forum/topic/332911-teclado-abnt2-microsoft-teclas-invertidas/?do=findComment&comment=2596673

**Comment by tiagodts** [comment]
Eu criei uma versão desse bundle que corrige a questão das teclas invertidas sem precisar alterar o tipo do teclado. Talvez seja útil para alguém que precise usar os dois tipos de teclado ABNT2 (ISO e ANSI) simultaneamente. Se esse não for o seu caso, continue seguindo a dica de @gabfr.
Se preferir manter o teclado configurado como ISO, baixe meu repo e instale a versão ISO. As duas versões podem ser instaladas simultaneamente: https://github.com/tiagodts/ABNT2-Layout

**Comment by ricardoreis** [comment]
Para resolver vá em **"Preferencias do Sistema..."** > clique no ícone  **"Teclado"** > depois clique na aba **"Teclado"** > depois clique no botão **"Alterar Tipo de Teclado..."**
